frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/components/CalendarView.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import interactionPlugin from "@fullcalendar/interaction";
+
+const MonthlyCalendar = () => {
+  const events = {
+    tenantId: "kl",
+    moduleName: "court",
+    slots: [
```

Comment:
this slot data I shared was slot meta data, it is for configuring how to group hearings after fetching

have this slot data as json, import into an api hook, and then use along with hearings fetch api hook ( this should be real api )